themes/default/content/docs/guides/pulumi-packages/schema.md

```diff
@@ -103,7 +103,6 @@ Complete schema examples that include a much wider range of schema configuration
 | Property            | Type                                       | Required | Description                                                                                                                       |
 |---------------------|--------------------------------------------|----------|-----------------------------------------------------------------------------------------------------------------------------------|
 | `name`              | `string`                                   | Yes      | Name is the unqualified name of the package (e.g. "aws", "azure", "gcp", "kubernetes", "random")                                  |
-| `displayName`       | `string`                                   | No       | The human-friendly name of the package.                                                                                           |
```

Comment:
@infin8x, was deleting these two table rows intentional?

Reply:
Hmm, nope - must've happened when I brought over my changes from the internal repo. Will add back.